tools/spanner/purge_ttl.py

```diff
@@ -36,21 +47,36 @@ def spanner_read_data(request=None):
     (instance_id, database_id) = from_env()
     instance = client.instance(instance_id)
     database = instance.database(database_id)
-    outputs = []
 
-    outputs.append("For {}:{}".format(instance_id, database_id))
+    logger.info("For {}:{}".format(instance_id, database_id))
+
     # Delete Batches. Also deletes child batch_bsos rows (INTERLEAVE
     # IN PARENT batches ON DELETE CASCADE)
+    batches_start = datetime.now()
     query = 'DELETE FROM batches WHERE expiry < CURRENT_TIMESTAMP()'
     result = database.execute_partitioned_dml(query)
-    outputs.append("batches: removed {} rows".format(result))
+    batches_end = datetime.now()
+    batches_duration = batches_end - batches_start
+    logger.info("batches: removed {} rows, batches_duration: {}".format(result, batches_duration))
+    statsd.timing("sync.purge_ttl.batches_duration", batches_duration)
```

Comment:
syncstorage-rs uses the "syncstorage" prefix"
```suggestion
    statsd.timing("syncstorage.purge_ttl.batches_duration", batches_duration)
```

Reply:
noted